sim/deathknight/deathknight.go

```diff
@@ -450,7 +448,7 @@ func (dk *Deathknight) CanCast(sim *core.Simulation, spell *core.Spell) bool {
 	case dk.BloodStrike:
 		return dk.CanBloodStrike(sim)
 	case dk.BloodBoil:
-		return dk.CanBloodBoil(sim)
+		return dk.BloodBoil.IsReady(sim)
```

Comment:
CanBloodBoil also checks for available cost, does IsReady check for that? I taught it does only cd checks